src/hexer/pipeline.nim

```diff
@@ -9,7 +9,30 @@
 
 include nifprelude
 
-import basics, iterinliner, desugar, xelim, duplifier, lifter, destroyer
+import ".." / nimony / [nimony_model, programs, decls]
+import basics, iterinliner, desugar, xelim, duplifier, lifter, destroyer, constparams
+
+proc publishHooks*(n: var Cursor) =
```

Comment:
Looks like an unrelated bugfix?

Reply:
It is needed because hooks are not in the index files, which cause problems when loading the symbol